app/services/api/v3/finance/statements_query.rb

```diff
@@ -23,6 +23,10 @@ def statements
               cohort_id: cohorts.map(&:id),
             )
 
+          if NpqApiEndpoint.disable_npq_endpoints?
```

Comment:
I wonder if we can use `statement_class` and only return `Finance::Statement::ECF` there instead to avoid the extra queries here might be simpler, similar to the declaration query?